src/web/components/signup/SignUp.tsx

```diff
@@ -1,9 +1,94 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { Container, Button, Typography } from '@material-ui/core';
+import '@fontsource/poppins';
+import { FormInput, FormInputPassword } from '../create-admin/FormInput';
 
-export const SignUp: React.FC = () => {
+const EmployeeSignup = () => {
     return (
-        <>
-            <p>SignUp component</p>
-        </>
+        <Box padding={0} margin={0}>
+            <Grid container spacing={0}>
+                <Grid item xs={6}>
+                    <Box style={{ margin: '20px 0 20px 100px' }}>
+                        <img src="/logo.png" width="100" height="100" alt="" />
+                    </Box>
+                    <Container
+                        style={{
+                            height: '100vh',
+                            backgroundColor: 'white',
+                            display: 'flex',
+                            margin: '50px 0 0 0'
+                        }}>
+                        <Grid item container direction="column" alignItems="center">
+                            <Typography
+                                style={{
+                                    color: '#000000',
+                                    fontWeight: 'bold',
+                                    fontSize: '30px',
+                                    letterSpacing: '4px'
+                                }}>
+                                Create Employee Account
+                            </Typography>
+                            <Typography style={{ color: '#867979' }}>
+                                Fill out the following information to create an Employee account
+                            </Typography>
+                            <br />
+                            <FormInput name="Company's Email" placeholder="contact@wuralab.com">
```

Comment:
The placeholder email should maybe reflect that this should for employee. Something like "employee@wuralab.com"